src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> These are all code pieces from `compat` where we assume the browser environment and don't support Node, browser extensions etc. All uses of `document` on `master` are via a module or a property of `window`, this is enforced via JSHint.

You misunderstand me. What I mean is that we create `<div>` elements multiple times during initialization (yes, [in](https://github.com/jquery/jquery/blob/0e4477c676db0427bb9b0bf39df8631501e62f24/src/css.js#L39) [master](https://github.com/jquery/jquery/blob/0e4477c676db0427bb9b0bf39df8631501e62f24/src/manipulation/support.js) [too](https://github.com/jquery/jquery/blob/0e4477c676db0427bb9b0bf39df8631501e62f24/src/css/support.js#L11)) with IIFEs that wouldn't even have to _be_ wrapped if they reused variables. I don't know the impact on file size, but I do know that those IIFEs survive minification and interrupt `var` consolidation. Some of which is properly laid at the feet of UglifyJS2, but there are still steps we could take to build a more compressible file. That's what I was referencing.